base/applications/rapps/include/defines.h

```diff
@@ -29,9 +29,18 @@
 #include "resource.h"
 #include "winmain.h"
 
-#define APPLICATION_DATABASE_URL L"https://rapps.reactos.org/rappmgr2.cab"
-#define APPLICATION_DATABASE_NAME L"rappmgr2.cab"
-#define MAX_STR_LEN              256
+/* Name of the registry sub-key where RAPPS settings are stored, and
+ * of the local AppData sub-directory where the RAPPS files are stored. */
+#define RAPPS_NAME                  L"RAPPS"
```

Comment:
I prefer `rapps` or `RApps`

Reply:
RApps looks like a typo to my eyes (although I know it maybe wasn't), what about capital R, small apps:
Rapps